### DISCUSSION TITLE
Sunbird Ed and Knowledge installation issue with GCP

### DISCUSSION THREAD

**Discussion by tsprasath** [Sunbird-Ed/Community#267] [Installation] [answered]
HI Sunbird,

I am setting up the sunbird Ed and Knowledge platform with sunbird-devops branch(Release-5.1.0).

Facing issue other user has faced earlier

https://github.com/orgs/Sunbird-Ed/discussions/76

Kindly need help..

**Comment by Krishnaj20** [comment, marked as answer]
@keshavprasadms @gandham-santhosh - FYA please

**Reply by beepdot** [reply]
@tsprasath Please raise individual discussion forum threads for each issue.
@sharathkashyap Please close this thread.
